parser_test.go

```diff
@@ -2216,10 +2227,13 @@ func TestParseModelAsTypeAlias(t *testing.T) {
                     "type": "string"
                 }
             }
+        },
+        "sql.NullInt64": {
+            "type": "object"
```

Comment:
Since sql.NullInt64 is not used directly anywhere (Params/Success/Failure) , i don't understand why it appears to definitions.

Declaring `type SQLNullInt64Alias sql.NullInt64` sould produce 
```
        "alias_external_types.SQLNullInt64Alias": {
            "$ref": "#/definitions/sql.NullInt64",
        },
        "sql.NullInt64": {
            "type": "object",
            "properties": {
                "Int64": {
                    "type": "integer"
                },
                "Valid": {
                    "type": "boolean"
                },
            }
        },
```
or at least 
```
        "alias_external_types.SQLNullInt64Alias": {
            "type": "object",
            "properties": {
                "Int64": {
                    "type": "integer"
                },
                "Valid": {
                    "type": "boolean"
                },
            }
        },
```